lib/salus/report.rb

```diff
@@ -189,7 +190,14 @@ def get_sarif_results(sarif_data)
     end
 
     def to_cyclonedx(config = {})
-      Cyclonedx::Report.new(@scan_reports, config).to_cyclonedx
+      cyclonedx_bom = Cyclonedx::Report.new(@scan_reports, config).to_cyclonedx
+      cyclonedx_report = {
+        autoCreate: true,
+        projectName: "#{@builds['org']}/#{@builds['project']}",
```

Comment:
`@builds` will only have value value if `builds` is defined in the salus config.  
If `builds` is not set, the code above will set `projectName` to `/`.

Perhaps you should just set projectName to `@project_name`.

Reply:
I might suggest you make this a configuration option.  That would allow the user to specify any value or combination of fields.  

For example:
```
  - uri: https://salus-config.internal2.net/salus-report
    format: cyclonedx-json
    put:
     cyclone_project_name:  "{{SALUS_BUILD_ORG}}/{{SALUS_BUILD_PROJECT}}"
      headers:
        Age: '12'
```

Reply:
Oh I see cyclonedx_options below... perfect place for this field IMHO

Reply:
By builds being defined in the salus config do you mean that the salus config contains the build param or if the {{SALUS_BUILD_ORG}} and {{SALUS_BUILD_PROJECT}} are set? I think in our case since we will be changing the "default" Salus scheduled scan yaml we'll be able to guarantee that builds params is set. My worry with using @project_name is that it specifies the project down to the commit level which will create a lot of outdated entries leading to false alarms for old code. Maybe we can use project name org and project are empty.

Reply:
@ghbren @joshua-ostrom Added in support for `cyclonedx_project_name` option in `cyclonedx_options`. If empty I default to empty string. Brianna mentioned that we could also default to `project_name` config var but users may want the flexibility to differentiate `project_name` from cyclonedx and have the cyclone project name as empty so left that out for now.